src/components/header/dropdownMenu.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import Select from 'react-select';
-import classnames from 'classnames';
 
 import { getTeams, getSelectedTeam } from 'actions/actions';
 import { COLORS } from 'enums';
+
+import * as Logos from 'assets/logos';
 import dropdownWhite from 'assets/icons/dropdownWhite.svg';
```

Comment:
These svg imports should always be after 3rd party imports and before local imports. Also all svg's should be CapitalizedCase:

```
import DropdownWhite from 'assets/icons/dropdownWhite.svg';
```

Actually, the correct name for this triangle is "Chevron"

```
import Chevron from 'assets/icons/chevron.svg
```

And while we're here, use svgs directly instead of putting it in an img src tag.

Change this `line 17`:

`  const dropdownIndicator = () => <img src={dropdownWhite} alt="dropdown arrow" />;`